plugins/com.google.cloud.tools.eclipse.util/META-INF/MANIFEST.MF

```diff
@@ -13,7 +13,6 @@ Export-Package: com.google.cloud.tools.eclipse.util,
  com.google.cloud.tools.eclipse.util.templates.appengine
 Import-Package: com.google.common.base;version="[20.0.0,21.0.0)",
  com.google.common.io;version="[20.0.0,21.0.0)",
- com.google.common.net;version="[20.0.0,21.0.0)",
```

Comment:
I get a compile error if I don't have this. (For you, I think removing this will increase the number of warnings?)